src/qt/button.cpp

```diff
@@ -46,7 +46,14 @@ bool wxButton::Create(wxWindow *parent, wxWindowID id,
 {     
     QtCreate(parent);
     SetLabel( wxIsStockID( id ) ? wxGetStockLabel( id ) : label );
-
+    if( style & wxBU_LEFT )
```

Comment:
Formatting should be consistent with one of
- wxW style which is `if ( condition )`;
- the only other `if` in this file, which is `if (condition)`.